employee-management/interface-document.md

```diff
@@ -177,20 +196,20 @@
 
 |HTTP Status|Error code|Error message|Description|
 |:----|:----|:----|:----|
-|400|invalid_date_of_birth_start|Date Of Birth is invalid.|The requested Employee's Birthday contains non-string characters. |
-|400|invalid_date_of_birth_end|Date Of Birth is invalid.|The requested Employee's Birthday contains non-string characters. |
-|400|invalid_salary_min|Employee salary min is invalid.|The requested Employee salary min is contains non-string characters. |
-|400|invalid_salary_max|Employee salary max is invalid.|The requested Employee salary max is contains non-string characters. |
-|404|not_found|Not found employee.|Not found employee information. |
+|400|invalid_date_of_birth_from|Date of birth is invalid.|The user enters a birthday that is not a valid date format.|
+|400|invalid_date_of_birth_to|Date of birth is invalid.|The user enters a birthday that is not a valid date format.|
+|400|invalid_salary_min|Employee salary is invalid.|Input argument of type 'char'|
+|400|invalid_salary_max|Employee salary is invalid.|Input argument of type 'char'|
+|404|not_found_employee_id|ID not found.|The user enters a non existent Employee ID.|
 
 ##### Sample Error Response
 
-- When a single error is used
+- When returning a single error
 
 ```json
 {
-  "code": "invalid_date_of_birth_start",
-  "message": "Date Of Birth is invalid."
+  "code": "invalid_date_of_birth_from",
+  "message": "Date of birth is invalid."
```

Comment:
`Date format is invalid.` thì sao?

Reply:
Em đã tìm hiểu thấy hợp lý và được sử dụng phổ biến trong error mesage. Em đã sửa lại Date format is invalid.